cluster/pulumi/canton-network/src/sv.ts

```diff
@@ -540,6 +541,7 @@ function installScan(
         }
       : {}),
     enablePostgresMetrics: true,
+    updateHistoryBackfillingValues,
```

Comment:
```suggestion
   ... updateHistoryBackfillingValues,
```
I think you want something like this